src/pathwayvis/components/info/info.component.html

```diff
@@ -19,10 +19,15 @@
       <md-table-container>
         <table md-table>
           <tbody md-body>
-            <tr md-row ng-repeat="change in ctrl.getMeasurements()">
+            <tr md-row ng-repeat="change in ctrl.getMeasurements()" ng-if="change.type == 'compound'">
```

Comment:
I'm not sure I understand what's going on here. Is it possible that in `ctrl.getMeasurements()` there are both compounds and non-compounds?

I think this logic should go to the controller because there's no way to catch errors in the template -> It's better to keep it as simple as possible.

What if you watch on `ctrl.getMeasurements()`?

    this.watch('ctrl.getMeasurements()', (measurements) => {
        this.measurements = measurements.map((change) => (change.type == 'compound' ?
            change :
            {
                  name: change.id,
                  units: {
                       numerator: 'mmol',
                       denominator: 'g',
                  },
                  rate: 'h',
            }
        ));
    });

And just display this in the template.

Even better I think the backend should return a response that is the same in both cases.